### PR TITLE
MM-52335: Enable non-transactional migrations

### PR DESCRIPTION
**Pull request by agnivade**
We introduce a new comment which will be detected by morph
before starting a migration. If that matches a special string,
then the migration won't be run inside a transaction.

https://mattermost.atlassian.net/browse/MM-52325

### CODE REVIEW

File: drivers/postgres/postgres.go [hunk 1]
```diff
@@ -208,33 +214,61 @@ func (pg *postgres) Apply(migration *models.Migration, saveVersion bool) (err er
...
+		if err != nil {
+			return &drivers.DatabaseError{
+				OrigErr: err,
+				Driver:  driverName,
+				Message: "error while committing a transaction to the database",
+				Command: "commit_transaction",
+			}
+		}
+	} else {
+		_, err := pg.conn.ExecContext(ctx, query)
```
Comment (isacikgoz) [inline drivers/postgres/postgres.go:250]: Maybe we can avoid the extra indent (suggestion may not compile but I like the UX :p)
```suggestion
		}
		
		return nil
	}

	_, err = pg.conn.ExecContext(ctx, query)
```

**Review by isacikgoz — APPROVED** [review]
Looks good to me, just a nit on styling.

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1:

File: drivers/postgres/postgres.go (hunk 1 above)
Comment (isacikgoz) [reply drivers/postgres/postgres.go:250]: 0/5 :p

**Pull request merged**